packages/storybook/src/nimble/table-column/mapping/table-column-mapping.mdx

```diff
@@ -25,35 +33,51 @@ The <Tag name={tableColumnMappingTag}/> renders specific number, boolean, or str
 <Controls of={tableColumnMappingStories.mappingColumn} />
 <ComponentApisLink />
 
-### Icon Mapping
+### Icon mapping
 
 <Description of={iconMappingStories} />
 <Controls of={iconMappingStories.iconMapping} />
 
-### Spinner Mapping
+### Spinner mapping
 
 <Description of={spinnerMappingStories} />
 <Controls of={spinnerMappingStories.spinnerMapping} />
 
-### Text Mapping
+### Text mapping
 
 <Description of={textMappingStories} />
 <Controls of={textMappingStories.textMapping} />
 
-### Empty Mapping
+### Empty mapping
 
 <Description of={emptyMappingStories} />
 <Controls of={emptyMappingStories.emptyMapping} />
 
 ## Usage
 
-### Best Practices
+### Best practices
+
+-   Provide a mapping for every expected record value. Because grouping is
+    performed on the record value, non-mapped record values can result in
+    multiple groups without group labels.
+    -   To improve grouping behavior of a value that shouldn't be rendered in
+        table cells, use a <Tag name={mappingEmptyTag}/> element to provide text
+        for that value's group row.
+-   Avoid having multiple values that map to the same text because grouping and
+    sorting is done on the record value rather than the mapped text.
+
+#### Choosing a mapping style
```

Comment:
For each place we refer to icon mappings in this guidance, it would be good to clarify that the guidance applies to spinner mappings too. If it's too wordy to include that everywhere, we could instead include a note at the beginning or end of the table, or even a separate row in the table. 

The subtlety is that "icon" is interchangeable with "spinner" for mappings but, I assume, NOT for headers. So maybe something like:

*Spinner mappings can be used in place of icon mappings throughout this guidance*

Reply:
Added explicit guidance not to mix and match. The spinner is more of a special case, since it augments the primary column mapping. Added a row to the table to explain that.